src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -459,6 +467,10 @@ private function createTypeAlternatives(ContainerBuilder $container, TypedRefere
         }
 
         $servicesAndAliases = $container->getServiceIds();
+        if (null !== ($autowiringAliases = $this->autowiringAliases[$type] ?? null) && !isset($autowiringAliases[''])) {
+            return sprintf(' Did you want to target one of "$%s"?', implode('", "$', $autowiringAliases));
```

Comment:
It will be helpfull indeed :)

What about telling instead: Available autowired services aliases are {$service1, $service2} for this {interface/class}

Of course it is longer, but it is more « precise » imho

Reply:
Works for me. Updated to `Available autowiring aliases for this %s are: "$%s".`.